sql/core/src/test/resources/sql-tests/inputs/postgreSQL/numeric.sql

```diff
@@ -884,6 +884,12 @@ SELECT '' AS to_char_10, to_char(val, 'S0999999999999999.999999999999999'), val
 -- SELECT '' AS to_char_35, to_char('100'::numeric, 'f"ool\"999');
 -- SELECT '' AS to_char_36, to_char('100'::numeric, 'f"ool\\"999');
 
+-- [SPARK-43815] Add support for additional SQL functions
```

Comment:
Please, change the ticket title. Now it is too much generic. The PR's title looks nice for that.